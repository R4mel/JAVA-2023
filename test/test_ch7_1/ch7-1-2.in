1
1
17
5
17
13
13
17
0
0

