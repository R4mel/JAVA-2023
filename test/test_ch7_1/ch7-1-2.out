
******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 17
[0] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[1] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[2] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[3] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[4] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[5] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 5
[0] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[1] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[2] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[3] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[4] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[5] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[6] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager
[7] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director
[8] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[9] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[10] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[11] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 17
[0] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[3] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[4] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director
[5] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager
[6] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[7] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[8] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[9] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[10] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[11] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 13
[0] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[2] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[3] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[4] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[5] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[6] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[7] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[8] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[9] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director
[10] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager
[11] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 13
[0] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director
[1] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager
[2] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[3] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:
[4] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[5] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[6] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[7] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[8] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[9] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[10] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[11] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 17
[0] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[1] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[2] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[3] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[4] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[6] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[7] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[8] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:
[9] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[10] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager
[11] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director

======================= Person Management Menu =======================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append 7.insert =
= 8.login 9.dispStudent(5_3) 10.dispPhone(5_3) 11.find(6_1)          =
= 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) 15.chAddress(6_2)  =
= 16.sort(7_1) 17.reverse(7_1) 18.binarySearch(7_1)                  =
======================================================================
menu item? 0

******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 0

Good bye!!
